external_server/server.py

```
@@ -0,0 +1,879 @@
+            if not self._running:
+                self._set_running_flag(True)
+            self._init_sequence()
+        except Exception as e:
+            self._log_and_set_error(e)
+            raise
+
+    def _run_normal_communication(self) -> None:
+        """Start the normal communication over MQTT. An init sequence must have been completed successfully."""
+        if self._state == ServerState.STOPPED:
+            return
+        elif not self.state == ServerState.INITIALIZED:
```

Ruff

Use `self.state != ServerState.INITIALIZED` instead of `not self.state == ServerState.INITIALIZED`

Replace with `!=` operator

(SIM201)